pkgs/by-name/be/beatprints/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "BeatPrints";
+  version = "1.1.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "TrueMyst";
+    repo = "BeatPrints";
+    rev = "v${version}";
+    hash = "sha256-HtYPEnHbJarSC3P337l3IGagk62FgEohSAVyv6PBnIs=";
+  };
+
+  build-system = with python3Packages; [
+    poetry-core
+  ];
+
+  pythonRelaxDeps = [
+    "Pillow"
```

```suggestion
    "Pillow"
    "rich"
```

On staging we have v14
```
Checking runtime dependencies for beatprints-1.1.4-py3-none-any.whl
  - rich<14.0.0,>=13.9.4 not satisfied by version 14.0.0
```